tests/virt_autotest/set_config_as_glue.pm

```diff
@@ -28,7 +28,8 @@ use testapi;
 
 sub fufill_guests_in_setting {
     my $wait_script        = "30";
-    my $get_vm_hostnames   = "virsh list  --all | grep sles | awk \'{print \$2}\'";
+    my $vm_host_types      = "sles|win";
```

Comment:
This var maybe interpreted differently by different people just from name. Suggest to use vm_types or anyone better.

Reply:
Done